meeshkan/__main__.py

```diff
@@ -69,30 +70,30 @@ def run_from_source(source: AbstractSource, sinks: Sequence[AbstractSink]):
 
 @click.command()
 @click.option("-i", "--input-file", required=False, type=click.File('rb'), help="Input file. Use dash '-' to read from stdin.")
+@click.option("-f", "--input-folder", required=False, type=click.Path(exists=True, file_okay=False, writable=True, readable=True), help="Input folder.")
```

Comment:
Would it maybe be simpler to just accept `--input` that can take either a file or a folder?

Reply:
Could be, although how would the typing work then? What I like about this is that there is a simple if/elif/elif.